docs/core/tools/cli-msbuild-architecture.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: .NET Core Command-line tools architecture
+title: Tools architecture
```

Comment:
I think command-line is important here.

Reply:
I removed it because .NET Core CLI is already added as a suffix. But I agree that "Tools architecture - .NET Core CLI" might not be enough.

```suggestion
title: Command-line tools architecture
```